### PR TITLE
feat: Issue Credential - adds wasm/rest js BDD tests

### LINKED ISSUE #1711: Issue Credential - need to add wasm/rest js BDD tests

### PR DESCRIPTION
**Pull request by soluchok**
Adds wasm/rest js BDD tests for the Issue Credential protocol.

* no more warnings
```
WARN [filelist]: All files matched by "/home/runner/work/aries-framework-go/aries-framework-go/test/aries-js-worker/test/common.js" were excluded or matched by prior matchers.
WARN [filelist]: All files matched by "/home/runner/work/aries-framework-go/aries-framework-go/test/aries-js-worker/test/environment.js" were excluded or matched by prior matchers.
```
* adds the possibility to run tests on the browser (see `karma.conf.js`)
* fixes error `timeout waiting for eve` by increasing timeout from 3 to 10 seconds
* `newDIDExchangeRESTClient` function works now but without the `router` (previously no one used it)
* fixes vdri tests 
* fixes verifiable tests
* adds wasm/rest js tests for the Issue Credential protocol

Closes #1711 
Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

File: test/aries-js-worker/test/issuecredential/issuecredential.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    for (let i = 0; i < retries; i++) {
+        try {
+            return await agent.verifiable.getCredentialByName({
+                name: name
+            })
+        } catch (e) {
+            console.warn(e.message)
+        }
+
+        await sleep(1000);
```
Comment (troyronda) [inline test/aries-js-worker/test/issuecredential/issuecredential.js:116]: sleeps make me worried about test fragility

Comment (soluchok) [reply test/aries-js-worker/test/issuecredential/issuecredential.js:116]: I don't know when the message will be saved. There is no way to listen to the post `StateMsg` event.  Or there is?

Comment (troyronda) [reply test/aries-js-worker/test/issuecredential/issuecredential.js:116]: I guess we should consider adding that capability :).

Comment (soluchok) [reply test/aries-js-worker/test/issuecredential/issuecredential.js:116]: I will check what we can do. Only the loop is the solution. Also, I saw we did it before.

Comment (troyronda) [reply test/aries-js-worker/test/issuecredential/issuecredential.js:116]: sure - loop will help.

File: cmd/aries-js-worker/main.go [hunk 2]
```diff
@@ -37,7 +37,7 @@ const (
 	ariesCommandPkg  = "aries"
 	ariesStartFn     = "Start"
 	ariesStopFn      = "Stop"
-	workers          = 2
+	workers          = 3
```
Comment (troyronda) [inline cmd/aries-js-worker/main.go:40]: 2 was failing?

Comment (soluchok) [reply cmd/aries-js-worker/main.go:40]: I saw it on master branch

Comment (soluchok) [reply cmd/aries-js-worker/main.go:40]: latest PR on master failed, I will check it again.

**Pull request merged**